modules/openapi-generator/src/main/resources/Java/libraries/google-api-client/ApiClient.mustache

```diff
@@ -53,6 +54,8 @@ public class ApiClient {
         module.addDeserializer(ZonedDateTime.class, CustomInstantDeserializer.ZONED_DATE_TIME);
         objectMapper.registerModule(module);
         {{/threetenbp}}
+        JsonNullableModule jnm = new JsonNullableModule();
```

Comment:
I don't think I saw this when I reviewed before. If JsonNullable provides a module which uses Jackson's APIs, then I think it will work the same across formats (json, xml, yaml).  That addresses the question I previously raised.

Reply:
FTR, I also made a PR for jackson-databind-nullable to document how to use JsonNullable with jackson's XML-serialization: https://github.com/OpenAPITools/jackson-databind-nullable/pull/11/files